tests/fixtures/config/config.yml

```diff
@@ -10,7 +10,7 @@ runtime:
 
     training:
         output_dir: test/training_output
-        auto_load_trained_model: true
+        save_with_id: true
```

Comment:
NIT: We shouldn't need this in the test fixture since it's the same value as the default